src/main/resources/synthea.properties

```diff
@@ -5,22 +5,22 @@ exporter.use_uuid_filenames = false
 exporter.subfolders_by_id_substring = false
 # number of years of history to keep in exported records, anything older than this may be filtered out
 # set years_of_history = 0 to skip filtering altogether and keep the entire history
-exporter.years_of_history = 10
```

Comment:
How do we want to treat this modification? I think the EXM130 changes, when merged in, will actually want a 5-10 year export history, because there's a ~10 year lookback on colonoscopies. I'm fine merging this in until then, with the understanding that from here on out we'll only increase this number, not decrease it.

Reply:
Good point

My ideas are that could either use branches here instead of trying to get everything into master, revert to patching in `fhir-bundle-calculator`, or try to combine the non-conflicting changes in master but keep patching in `fhir-bundle-calculator` as well.

@tstrass thoughts?

Reply:
Is there any real downside to just using whatever ends up being the max years of history needed by any individual measure? I'm thinking it might take longer to generate/calculate, but probably not prohibitively longer? Does using 10 years instead of 2 here make it a lot less likely to get patients to measure populations?

Reply:
> Does using 10 years instead of 2 here make it a lot less likely to get patients into measure populations?

Yes for the current exm124 module.

The calculation time doesnt seem to be significantly different between 2/10, but the population hit rates do drop due to not hitting the IPOP:

```
ValidEncounter
		where ValidEncounter.period during day of MeasurementPeriod
		and ValidEncounter.status  = 'finished' 
```

Changing to `> 2` will probably drop the hit rates for at least EXM125 as well as they share that clause in IPOP.

Reply:
As far as I know, using 10 years here doesn't change measure population likelihood for this measure--it just makes there be ~5x as much info in the FHIR patient bundles.

Reply:
@losborne I'm curious how that will affect measure hit rates. Won't there just be more encounters to test for that `ValidEncounter` status?

Reply:
I'm going to change this to 10.  If the reviewers determine this causes things to no longer be useful, I will add the `patch` back in to `fhir-bundle-calculator` to set it to 2, or we can decide to re-write the module to be custom like exm130.